prime-router/src/main/kotlin/SettingsProvider.kt

```diff
@@ -52,5 +52,11 @@ enum class Topic(val json_val: String) {
     COVID_19("covid-19"),
 
     @JsonProperty("monkeypox")
-    MONKEYPOX("monkeypox")
+    MONKEYPOX("monkeypox"),
+
+    @JsonProperty("CsvFileTests-topic")
```

Comment:
I wonder if this is even needed.  Can you ask someone like Jim or Mo?

Reply:
@MauriceReeves-usds Any thoughts on this? In short, do we need/want to keep separate default filters and tests for topic `"CsvFileTests-topic"`?

Reply:
I am not sure on this. This might be related to the CSV uploader, which means we may not need it now. Go ahead and leave it in, but make a tech debt ticket to investigate if we need this topic, and if we don't, remove it. That way it's tracked, and can be worked, but the rest of your changes can move forward.

Reply:
Sounds good, will do, thanks!